src/Illuminate/Routing/RouteCollection.php

```diff
@@ -261,6 +261,17 @@ public function hasNamedRoute($name)
     }
 
     /**
+     * Get a route instance by its method.
+     *
+     * @param  string  $method
+     * @return \Illuminate\Routing\Route|null
+     */
+    public function getByMethod($method)
+    {
+        return isset($this->routes[$method]) ? $this->routes[$method] : null;
```

Comment:
@JosephSilber I followed the same style as the lines [282](https://github.com/laravel/framework/pull/10721/files#diff-6200babcb528ffc5681be487bf9446f2R282) ans [293](https://github.com/laravel/framework/pull/10721/files#diff-6200babcb528ffc5681be487bf9446f2R293).